src/app.js

```diff
@@ -2,21 +2,36 @@ import express from 'express';
 import fs from 'fs';
 import { urlencoded, json } from 'body-parser';
 import cors from 'cors';
+import passport from 'passport';
+import session from 'express-session';
 import morganLogger from 'morgan';
 import router from './routes/index';
 import logger from './utils/winston';
-import Responses from './utils/response';
+import Responses from './utils/Response';
 import config from './config';
 import ErrorHandler from './utils/error';
 
 const isDevelopment = config.env;
 
 const app = express();
 
+app.use(session({
+  secret: process.env.SECRET,
+  resave: false,
+  saveUninitialized: true,
+  cookie: { secure: false },
+}));
```

Comment:
Why session?

Reply:
I used `catchErrors` function instead to keep our test coverage.